plugins/audio_file_processor/audio_file_processor.cpp

```diff
@@ -1063,11 +1069,7 @@ void AudioFileProcessorWaveView::updateGraph()
 		m_to = m_sampleBuffer.endFrame();
 	}
 
-	if( m_sampleBuffer.reversed() != m_reversed )
-	{
-		reverse();
-	}
-	else if( m_last_from == m_from && m_last_to == m_to && m_sampleBuffer.amplification() == m_last_amp )
+	if( m_last_from == m_from && m_last_to == m_to && m_sampleBuffer.amplification() == m_last_amp )
```

Comment:
This won't work correctly even after fixing `reverseModelChanged` if the start point is at the very beginning and the end point is at the very end(`m_from` is 0 and `m_to` is equal to `m_sampleBuffer.frames()`).